packages/lib-react-components/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unpublished]
+- Allow for modals that can't be closed. [#1983]
```

Comment:
Could this be placed under the `Fixed` sub-heading?

Reply:
Sure!